app/javascript/app/phone-internationalization.js

```diff
@@ -72,12 +72,16 @@ const updateOTPDeliveryMethods = () => {
     phoneLabel.classList.add('btn-disabled');
     smsRadio.click();
     deliveryMethodHint.innerText = warningMessage;
-    optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info_modile_only');
+    if (optPhoneLabelInfo) {
+      optPhoneLabelInfo.innerText = I18n.t('devise.two_factor_authentication.otp_phone_label_info_modile_only');
+    }
```

Comment:
Did you mean "mobile" instead of "modile"?

Reply:
in the code it was already this way, I can certainly update the application to correct this though